.gitignore

```diff
@@ -1,3 +1,4 @@
+*.a
```

Comment:
This project does not support CMake in-source build. You will never see these if the build directory is called "build." Please remove this commit.